ng2-components/ng2-activiti-form/src/components/widgets/typeahead/typeahead.widget.spec.ts

```diff
@@ -369,17 +369,15 @@ describe('TypeaheadWidgetComponent', () => {
                     id: 'typeahead-id',
                     name: 'typeahead-name',
                     type: 'readonly',
-                    value: '9',
+                    value: 'F',
                     params: { field: { name: 'typeahead-name', type: 'typeahead' } }
                 });
                 fixture.detectChanges();
-                const trigger = fixture.debugElement.query(By.css('.mat-select-trigger')).nativeElement;
-                trigger.click();
-                fixture.detectChanges();
                 fixture.whenStable().then(() => {
+                    fixture.detectChanges();
                     expect(element.querySelector('#typeahead-id')).not.toBeNull();
-                    let optionElement: HTMLElement = <HTMLElement> document.body.querySelector('.mat-option');
-                    expect(optionElement.innerText).toEqual('FakeProcessValue');
+                    let inputElement: HTMLInputElement = <HTMLInputElement> element.querySelector('#typeahead-id');
```

Comment:
With this expectation change, the test name doesn't not reflect what the test really asserts.